### ISSUE TITLE
Footer

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/genepanel.iobio#271]
Should we have a standard footer on these apps? Often this is where we would have contact us links etc.

@tonydisera, what do you think?

**Comment by tonydisera** [comment]
Yes, I think that would be a good idea.  We are short on vertical space in our apps, so that would be the only drawback.  Did you guys come up with a nice footer for Mosaic?

**Comment by AlistairNWard** [comment]
We have a very simple footer, but I think that's enough. 

![Mosaic.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5750332babe50b1300c53bf6/125516c66e82c728ace21e0d46c59b89228b80ebc3abaf51ab57f5973d047d317c05fa2e74dcba2aea447d510d5a4eac400a5d47adf18b7fb4aa2e32db025fb28c341521842f1bbeb98a15be04b9268a4781b6b98844c0e12f68.png)

**Comment by tonydisera** [comment]
That is cool.  I like that it doesn't dominate the UI.  We should do this in our lab apps.

**Comment by adityaekawade** [comment]
I have currently added the footer in the overview page. Should it be in every component or just the homepage?

**Comment by AlistairNWard** [comment]
Yes, this should be a consistent element across all pages.

**Comment by adityaekawade** [comment]
I have added the footer with few links that is consistent across all the pages. Let me know the text and the links that need to go in there.
